core/src/main/java/org/opensearch/sql/expression/ReferenceExpression.java

```diff
@@ -100,6 +104,12 @@ public ExprValue resolve(ExprTupleValue value) {
   }
 
   private ExprValue resolve(ExprValue value, List<String> paths) {
+    // This case is to allow returning all values in an array to be in one row
+    if (value.type().equals(ExprCoreType.ARRAY)){
+      return new ExprCollectionValue(value.collectionValue().stream()
```

Comment:
Looks like code coverage is complaining about no test for when the `if` statement is true?